src/Form/Type/PhoneNumberType.php

```diff
@@ -142,6 +142,7 @@ public function configureOptions(OptionsResolver $resolver): void
             'preferred_country_choices' => [],
             'country_options' => [],
             'number_options' => [],
+            'manage_leading_zeros' => false,
```

Comment:
Could you add a comment explaining what this options does please? With a `@see` to point to the issue to give more context that could be perfect. :)

Reply:
Tell me if you agree with my description :)

Reply:
That's perfectly clear! :ok_hand: